tests/order/order.variables.js

```diff
@@ -34,144 +27,254 @@ const newOrder = {
+        { lang: 'en', value: 'Cordura fabric' },
       ],
-      additions: [],
-      actualPrice: [
-        {
-          currency: 'UAH',
-          value: 90000,
-        },
+      bottomColor: colorId,
+      availableCount: 777,
+      additions: [
         {
-          currency: 'USD',
-          value: 3246,
+          available: true,
+          name: [
+            { lang: 'uk', value: 'Кишеня' },
+            { lang: 'en', value: 'Pocket' },
+          ],
+          description: [
+            { lang: 'uk', value: 'Бокова кишенька за бажанням' },
+            { lang: 'en', value: 'Side pocket by request' },
+          ],
+          additionalPrice: [
+            { currency: 'UAH', value: 145000 },
+            { currency: 'USD', value: 5229 },
+          ],
         },
       ],
-      quantity: 1,
     },
   ],
-  paymentMethod: 'CARD',
+});
+const newClosure = materialId => ({
+  name: [
+    { lang: 'uk', value: ' тест' },
+    { lang: 'en', value: ' test' },
+  ],
+  material: materialId,
+  available: true,
+});
+const newPattern = {
+  name: [
+    { lang: 'uk', value: 'Паттерн тест' },
+    { lang: 'en', value: 'Pattern test' },
+  ],
+  description: [
+    { lang: 'uk', value: 'Опис тест' },
+    { lang: 'en', value: 'Description test' },
+  ],
+  material: ' material',
 };
-
-const newOrderMutation = {
-  ...newOrder,
-  status: 'SENT',
+const newModelInputData = (categoryId, constructorId) => ({
+  category: categoryId,
+  name: [
+    {
+      lang: 'ua',
+      value: 'Тестова модель',
+    },
+    {
+      lang: 'en',
+      value: 'Тестова модель',
+    },
+  ],
+  description: [
+    {
+      lang: 'ua',
+      value: 'Тестовий опис моделі',
+    },
+    {
+      lang: 'en',
+      value: 'Test description',
+    },
+  ],
+  priority: 1,
+  constructorBasic: [constructorId],
+});
+const newCategoryInputData = {
+  code: 'Code for test',
+  name: [
+    { lang: 'uk', value: 'Категорія тест' },
+    { lang: 'en', value: 'Category test' },
+  ],
+  images: {
+    large: 'large_10b1e9gkhm7xa9m_backpacks.jpg',
+    medium: 'medium_10b1e9gkhm7xa9m_backpacks.jpg',
+    small: 'small_10b1e9gkhm7xa9m_backpacks.jpg',
+    thumbnail: 'thumbnail_10b1e9gkhm7xa9m_backpacks.jpg',
+  },
+  available: true,
 };
-
-const updatedData = {
+const newMaterialInputData = colorId => ({
+  name: [
+    { lang: 'uk', value: 'Матеріал тест' },
+    { lang: 'en', value: 'Material test' },
+  ],
+  colors: [colorId],
+  description: [
+    { lang: 'uk', value: 'Опис update' },
+    { lang: 'en', value: 'Description update' },
+  ],
+  available: true,
+  additionalPrice: 100,
+});
+const newColorInputData = {
+  name: [
+    { lang: 'uk', value: 'Тестовий колір test' },
+    { lang: 'en', value: 'Test color test' },
+  ],
+  colorHex: 'colorHex test',
+  simpleName: [
+    { lang: 'uk', value: 'Проста назва кольору test' },
+    { lang: 'en', value: 'Simple color name test' },
+  ],
+};
+const newConstructorBasic = (materialId, colorId) => ({
```

Comment:
I think it would be better to move this file func in separate file inside constructor folder